Source/Core/DiscIO/Blob.cpp

```diff
@@ -177,23 +178,24 @@ std::unique_ptr<IBlobReader> CreateBlobReader(const std::string& filename)
   if (cdio_is_cdrom(filename))
     return DriveReader::Create(filename);
 
-  if (!File::Exists(filename))
+  File::IOFile file(filename, "rb");
+  u32 magic;
```

Comment:
Would it be too nitpicky to request this variable (and subsequent "magic" vars) be named something more descriptive?

magic_num or something. If you disagree, feel free to ignore. Not terribly important.